src/dipdup/cli.py

```diff
@@ -17,6 +17,7 @@
 
 import click
 import uvloop
+from aerich.cli import cli as aerich_cli
```

Comment:
This import should be lazy, since it takes 0.6s and  `dipdup.cli` module is imported for every command.